wire-ios/Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/EstablishingCallStatusView.swift

```diff
@@ -40,7 +40,7 @@ extension CallStatusViewState {
         switch self {
         case .none: return ""
         case .connecting: return CallStatus.connecting
-        case .ringingIncoming(name: let name?): return CallStatus.Incoming.user("\(name ?? "")")
+        case .ringingIncoming(name: let name?): return CallStatus.Incoming.user("\(name)")
```

Comment:
Is "name" not optional?

Reply:
No! In that case I fixed this error:

Left side of nil coalescing operator '??' has non-optional type 'String', so the right side is never used